tooling/github-strategy.md

```diff
@@ -0,0 +1,92 @@
+| --- | --- | --- | --- | --- | --- | --- |
+| Learning, copy and pasting (examples, templates)​ | Learning​ | None​ | No​ | Most​ | Entropy and lost time.​ | Probably low​ |
+| Use Reusable tools built for Company circumstances. ​ | Reuse​ | None-little​ | No​ | Many​ | Loss of efficiency and speed. ​ | Probably low​ |
+| Content change, fix, add, or update (websites, documentation)​ | Contribution, small ones​ | None-little​ | No​ | Many​ | Wasted time due to bad information.​ | Probably low​ |
+| Build within someone else's internal service | Contribution, within well-defined parameters​ | Yes​ | No​ | Some​ | More things never done at all.​ | Probably low but more exceptions.​ |
+| Deduplication. Don't built same thing twice, build general solution.​ | Contribution ​ | Yes​ | Varies​ | Some​ | General purpose code written less often. ​ | Depends, some very high.​ |
+| Ensure alignment between related projects​ | Contribution ​ | Yes​ | Likely​ | A few​ | Loss of efficiency and speed.​ | Depends, some very high.​ |
+| Not get slowed down by who is owning ​ | Contribution ​ | Yes​ | Likely​ | A few​ | Loss of efficiency and speed.​ | Depends, some very high.​ |
+
+## Project Participation Difficulty Level by Setting
+
+The difficulty level to join a project depends on your access rights to GitHub Enterprise and the repository type. It should be noted that GitHub settings in the enterprise often use SCIM and work with IdPs such as Microsoft Entra and Okta, but the participation hurdle can also vary depending on the settings on the IdP side.
+
+Here are groups that can be used to define repository permissions ordered by process toil to join:
+
+1. Everyone behind company firewall
+1. Internal visibility​ (everyone on platform is in this group)
+1. Security group created automatically​
+1. Security group with self-join and NO forced renew after X time period​
+1. Security group with self-join and forced renew after X time period​
+1. Security group with self-initiated but requires waiting for manager permission.
+1. Security group with that requires emailing someone to manually add you
+
+## Pros and Cons of InnerSource Dedicated Environment
+
+Although InnerSource catalogs and organizations specifically for InnerSource have benefits, they should not be viewed as a place that holds “ALL” InnerSource.
+
+| Approach​ | Definition​ | Advantages​ | Disadvantages​ |
+| --- | --- | --- | --- |
+| InnerSource Catalogs | Repositories that want extra visibility self-select via topic tag. They are then harvested into a separate catalog.​ | Have some of the repos most interested in being reused in one place helps measurement & discovery of them.​ | Most users won’t troll a catalog separate from their workflow.​ There is a risk a perception is created that only repos in that catalog are innerSource.​ |
```

Comment:
From @dellagustin  ([The original review comment](https://docs.google.com/document/d/1lPR90dIiWKlsNlV46xhGkR9pP0GNYmeAB7xOk-_MAYo/edit?disco=AAAA1q2rBcA))

Consider using additionally the term portal and linking to the pattern [https://patterns.innersourcecommons.org/p/innersource-portal](https://www.google.com/url?q=https://patterns.innersourcecommons.org/p/innersource-portal&sa=D&source=docs&ust=1691047026248081&usg=AOvVaw3hwV-cgqrhRNfpaQ_lJQzm)